src/components/ActionItem/ActionItem.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Image } from 'react-native';
+import { string, bool, number } from 'prop-types';
+import Tick from 'assets/images/tickIcon/default.png';
+
+import BulletItem from 'components/BulletItem';
+
+const ActionItem = ({ id, description, completed }) => {
+  return (
+    <BulletItem isOdd={id % 2 === 0} text={description}>
+      {completed && <Image source={Tick} />}
+    </BulletItem>
+  );
+};
```

Comment:
Estan pasandole un index a una prop id, y la estan usando como index. Podreian cambiar el nombre de la prop, pero en mi opinión sería mejor parale una prop isOdd.

```suggestion
const ActionItem = ({ isOdd, description, completed }) => (
    <BulletItem isOdd={isOdd} text={description}>
      {completed && <Image source={Tick} />}
    </BulletItem>
  );
```